envsec/internal/jetcloud/git.go

```diff
@@ -0,0 +1,31 @@
+package jetcloud
+
+import (
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+func createGitIgnore(wd string) error {
+	gitIgnorePath := filepath.Join(wd, dirName, ".gitignore")
+	return os.WriteFile(gitIgnorePath, []byte("*"), 0600)
+}
+
+func gitRepoURL(wd string) (string, error) {
+	cmd := exec.Command("git", "config", "--get", "remote.origin.url")
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		return "", err
```

Comment:
What happens if the user does not have `git` installed?

Reply:
if no git, we ignore errors and we pass empty strings for both repo and subdirectory. In this case the API should return a new ID.